src/chainparams.h

```diff
@@ -24,6 +23,14 @@ struct SeedSpec6 {
     uint16_t port;
 };
 
+typedef std::map<int, uint256> MapCheckpoints;
+
+struct CCheckpointData {
+    MapCheckpoints mapCheckpoints;
+    int64_t nTimeLastCheckpoint;
+    int64_t nTransactionsLastCheckpoint;
+    double fTransactionsPerDay;
```

Comment:
ACK on the suggestion, but I'm going to defer it to a subsequent PR, because this code is later altered upstream by https://github.com/bitcoin/bitcoin/pull/9472, so doing this afterwards means we can easily push that patch upstream too.

Reply:
Would be nice to initialize.  Upstream don't initialize in the subsequent change so any initialization changes then will still impact potential future backports.

Reply:
Hmm, it turns out that [due to a restriction added in C++11](https://stackoverflow.com/a/39344592) (and removed in C++14, presumably because it annoyed everyone), this does not work. Specifically, if we add default member initializers, we can't use aggregate initialization. The latter is important for readability of `chainparams.cpp`, so I am now NACK on this while we are on C++11.